includes/class-sensei-frontend.php

```diff
@@ -1498,102 +1498,13 @@ public function remove_active_course( $order_id ) {
 	 * @return void
 	 */
 	public function activate_purchased_courses( $user_id = 0 ) {
+		_deprecated_function( __METHOD__, '2.0.0', '\Sensei_WC_Paid_Courses\Courses::activate_purchased_courses' );
```

Comment:
We seem to be taking 2 different approaches to deprecating functions. Always showing a deprecation notice as we're doing here, or only showing a deprecation notice if the corresponding function is not available in WCPC as per [this code](https://github.com/Automattic/sensei/pull/2413/commits/66a916214ebeb3937584e3c128ae690c07dfe795#diff-978209e01c6c9e4491c170874c427461R163).

It would be good to standardize and choose an approach. I don't have a preference as to which to use, only that we're consistent.